tactics/class_tactics.ml

```diff
@@ -373,15 +372,17 @@ and e_my_find_search db_list local_db secvars hdc complete only_classes env sigm
       in
       let tac = FullHint.run h tac in
       let tac = if complete then Tacticals.tclCOMPLETE tac else tac in
-      let pp =
+      let pp() =
```

Comment:
This does not really matter because literally nobody uses filtered unification. It's broken beyond any potential use case.